ietf/secr/rolodex/views.py

```diff
@@ -91,10 +90,10 @@ def add_proceed(request):
                              )
 
             # in theory a user record could exist which wasn't associated with a Person
-            try:
+            
+            user = User.objects.filter(username__iexact=email).first()
+            if not user:
```

Comment:
Restrict username to lowercase like we do in the main `create_account()` view?

Reply:
Probably a good idea. I've reached out to staff to see if this feature is still used. If not it should be removed.

Reply:
Not sure if maybe you meant to point to L96?. The code you point to is looking to see if a User already exists, and _that_ needs to use `__iexact`. I could see at 96 forcing an email.lower() for the username argument, which should be made explicit instead of positional.

Is that what you had in mind?

Reply:
Yes, was pointing to the block rather than the specific line (github sometimes is picky about which lines you can comment on).

I guess since we already have some mixed case usernames it's not essential, but I think we should be consistent about whether we're forcing lowercase usernames or not.

Reply:
I'll solve this in a slightly different way - this block gets `email` from NewPersonForm, and that form should include downcasing email while cleaning it.